### PR TITLE
navigation-component

### PR DESCRIPTION
**Pull request by bipin7yadav**
-navigation bar

### CODE REVIEW

File: Navigation.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+header{
+    display: flex;
+    justify-content: space-between;
+    margin: 5px;
+    background-color: var(--secondary-color);
+    border-bottom-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+    padding:1px;
+    top: 0;
```
Comment (kiranpratyush) [inline Navigation.css:32]: We can remove it

**Review by kiranpratyush — APPROVED** [review]

File: Navigation.css (hunk 1 above)
Comment (bipin7yadav) [reply Navigation.css:32]: remove What?

Comment (kiranpratyush) [reply Navigation.css:32]: Sorry, we can remove top : 0 because it has no effect in the flex layout.

File: Navigation.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    position: relative;
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    margin: 5px;
+    background-color: var(--secondary-color);
+    border-bottom-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
```
Comment (mimansha-swarup) [inline Navigation.css:30]: suggestion: you can use shorthand property too
image

File: Navigation.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+.side-nav:hover{
+    background-color: var(--primary-color);
+    padding: 5px 3rem 5px 3rem;
+    border-radius: 1rem;
+}
+
+
+.sidebar{
+    width: 16rem;
+    height: fi;
```
Comment (mimansha-swarup) [inline Navigation.css:48]: typo: i guess it should be fit-content

File: Navigation.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    width: 100%;
+}
+
+.body{
+    width: 100%;
+}
+.container{
+    padding: 3px;
+    display: flex;
+    gap: 5px;
```
Comment (mimansha-swarup) [inline Navigation.css:20]: i will suggest not to use px you can use rem or em

**Review by mimansha-swarup — COMMENTED** [review]
few suggestion

File: Navigation.css (hunk 4 above)
Comment (bipin7yadav) [reply Navigation.css:20]: but we cannot use rem when unit is below 10 px

**Pull request merged**